x/feeabstraction/ante/evm/mono_decorator_test.go

```diff
@@ -110,6 +110,10 @@ func TestMonoDecorator(t *testing.T) {
 	err := app.FeeMarketKeeper.SetParams(ctx, feeMarketParams)
 	require.NoError(t, err)
 
+	// Reactivate keeper since it was disabled on abci
+	err = app.FeeAbstractionKeeper.Params.Set(ctx, types.DefaultParams())
+	require.NoError(t, err)
```

Comment:
Same as https://github.com/KiiChain/kiichain/pull/164/files#r2515152069. Why set this as the default here?

The module auto-disables itself on setup?